utils/data/standard_vars.csv

```diff
@@ -0,0 +1,76 @@
+55;7;2;Yes;LWdown;surface_downwelling_longwave_flux_in_air;W/m2;Surface incident longwave radiation;1;Driver;No;Yes;real;3;lon;lat;time;na;Surface incident longwave radiation
+56;7;3;Yes;Psurf;air_pressure;Pa;Surface pressure;1;Driver;No;Yes;real;3;lon;lat;time;na;Surface pressure
+57;7;4;Yes;Qair;specific_humidity;kg kg-1;Near surface specific humidity;1;Driver;No;Yes;real;3;lon;lat;time;na;Near surface specific humidity
+58;7;5;Yes;Rainf;precipitation_flux;kg m-2 s-1;Rainfall rate;1;Driver;No;Yes;real;3;lon;lat;time;na;Rainfall rate
+59;7;6;Yes;SWdown;surface_downwelling_shortwave_flux_in_air;W m-2;Surface incident shortwave radiation;1;Driver;No;Yes;real;3;lon;lat;time;na;Surface incident shortwave radiation
+60;7;7;Yes;Tair;air_temperature;K;Near surface air temperature;1;Driver;No;Yes;real;3;lon;lat;time;na;Near surface air temperature
+61;7;8;Yes;Wind;wind_speed;m s-1;Near surface module of the wind;1;Driver;No;Yes;real;3;lon;lat;time;na;Near surface wind magnitude
+62;7;9;Yes;Tmin;air_temperature_max;K;Daily Maximum Temperature;1;Driver;No;Yes;real;3;lon;lat;time;na;Daily Maximum Temperature
+63;7;10;Yes;Tmax;air_temperature_min;K;Daily Minimum Temperature;1;Driver;No;Yes;real;3;lon;lat;time;na;Daily Minimum Temperature
+64;7;11;Yes;Uwind;northward_wind;m s-1;Northward Component of Wind;1;Driver;No;Yes;real;3;lon;lat;time;na;Northward Component of Wind
+65;7;12;Yes;Vwind;eastward_wind;m s-1;Eastward Component of Wind;1;Driver;No;Yes;real;3;lon;lat;time;na;Eastward Component of Wind
+66;7;13;Yes;RH;relative_humidity;%;Relative Humidity;1;Driver;No;Yes;real;3;lon;lat;time;na;Relative Humidity
+67;7;14;Yes;PAR;surface_downwelling_photosynthetic_photon_flux_in_air;mol m-2 s-1;Photosynthetically Active Radiation;1;Driver;No;Yes;real;3;lon;lat;time;na;Photosynthetically Active Radiation
+68;7;15;Yes;precipf;;kg m-2 s-1;Precipitation;1;Driver;Yes;Yes;real;3;lon;lat;time;na;"The per unit area and time precipitation representing the sum of convective rainfall, stratiform rainfall, and snowfall"
+69;8;1;Yes;BA;;m2 ha-1;Basal area;1;Diversity;No;Yes;real;4;lon;lat;time;PFT;Basal area by PFT
+70;8;2;Yes;Dens;;1 ha-1;Stem Density;1;Diversity;No;Yes;real;4;lon;lat;time;PFT;Stem Density dy PFT
+71;8;3;Yes;Fcomp;;kgC kgC-1;Aboveground Biomass Fractional Composition;1;Diversity;No;Yes;real;4;lon;lat;time;PFT;Aboveground biomass Fractional composition of each PFT within each grid cell
+72;8;4;Yes;PFT;;(-);Plant Functional Type;1;Diversity;No;Yes;character;1;PFT;na;na;na;Name of each plant function type or species included in the model
+73;8;5;Yes;Estab;;1 ha-1;New Individuals;1;Diversity;No;Yes;real;4;lon;lat;time;PFT;New Individuals
+74;8;6;Yes;Mort;;1 ha-1;Mortality;1;Diversity;No;Yes;real;4;lon;lat;time;PFT;Individuals lost through death
+75;9;1;Yes;SoilDepth;;m;Soil Depth Layer;1;other;No;no;real;1;nsoil;na;na;na;Depth to the bottom of each model-defined soil layer
```

Comment:
Well, I don't recall the exact discussion that resulted in adding SoilDepth to the PalEON standard, but that's one that we can't just add here since it creates two variables (SoilDepth and z_bottom) that are identically defined. I suspect the PalEON team realized what we've been discussing: 

1. That z_top, z_bottom, and z_node are pretty redundant
2. That something with 'depth' in the name makes more sense to most people

However, as you note, 'soil_depth' in BETY is the total depth, so we don't want to use that. Also unfortunately, the BETY variable 'depth' has units of cm and will have existing records as a covariate (and possibly code) that would need conversion if we switch the units to m.

I think we should do the switch on depth's units, but as an interim stopgap I propose that we 
1. follow the PalEON change of dropping the z_* variables
2. that we retain the PalEON SoilDepth variable BUT
3. we instead have SoilDepth be a dimension 
4. we explicitly flag depth and SoilDepth as redundant
5. we don't forget to go back and update depth's units and then migrate all SoilDepth records/code/etc to be depth

If we're going to make this change it will require a quick grep of the model modules to update any models outputting z_* variables.  That said, the top gap will ultimately be a pain to fix so it might just be simpler to bite-the-bullet and fix `depth` right now while we're at it.